app/views/coronavirus_landing_page/components/landing_page/_postcode-lookup.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+  input_label ||= "Enter your postcode"
+  input_value ||= ""
+  input_error ||= ""
+  hint_text ||= "For example SW1A 2AA"
+  button_text ||= "Find"
+  error_message ||= nil
+  error_description ||= nil
+  autocomplete ||= "postcode"
+  link ||= { text: "Find a postcode on Royal Mail's postcode finder", href: "http://www.royalmail.com/find-a-postcode"}
+%>
+<% if error_message %>
+  <%= render "govuk_publishing_components/components/error_alert", {
+      message: error_message,
+      description: error_description,
+    } %>
+<% end %>
+<div class="covid__postcode-lookup">
+  <form action="" method="post">
+    <%= render "govuk_publishing_components/components/input", {
+          label: {
+            text: input_label
+          },
+          heading_size: "s",
+          error_message: input_error,
+          value: input_value,
+          name: "covid__postcode-lookup",
+          hint: hint_text,
+          invalid: error_message ? 'true' : 'false',
+          autocomplete: autocomplete,
```

Comment:
```suggestion
          autocomplete: "postal-code",
```

Reply:
Probably makes more sense to keep as the `autocomplete` as a var with the default being updated, equally happy to remove var - as this is an MVP partial (eg this is unlikely to not be anything but a postcode input)

Reply:
Updated to `autocomplete ||= "postal-code"` - `postal-code` fallback but can be passed in a different value if needed